<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QDialog" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Automatic Backup</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QListWidget" name="backupList">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QPushButton" name="newBackupButt">
         <property name="toolTip">
          <string>Create a new backup configuration.</string>
         </property>
         <property name="text">
          <string>New...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editBackupButt">
         <property name="toolTip">
          <string>Open the properties of the selected configuration.
If more than one are selected, execute them.</string>
         </property>
         <property name="text">
          <string>Select...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteBackupButt">
         <property name="toolTip">
          <string>Delete the selected backup configuration.</string>
         </property>
         <property name="text">
          <string>Delete...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cleanButt">
         <property name="toolTip">
          <string>Analyze a given directory and show a tree with
all direcrtories and files older than a given date.
Clean up will be possible with a context menu.</string>
         </property>
         <property name="text">
          <string>Clean up...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="keepRun">
         <property name="toolTip">
          <string>If checked, the backup tool will keep running
upon Exit. It will show a little icon and
periodically check all backup configurations
for automatic execution.</string>
         </property>
         <property name="text">
          <string>Keep running</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="shutDown">
         <property name="toolTip">
          <string>If checked, the backup tool context menu
will allow you to execute all pending
backups and to initiate a shut down.
In this mode, no periodically check is done.</string>
         </property>
         <property name="text">
          <string>Shut down menu</string>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButt">
         <property name="toolTip">
          <string>Exit the tool or closes the main dialog if
Keep running is checked.</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="settingstButton">
         <property name="toolTip">
          <string>program information and global settings</string>
         </property>
         <property name="text">
          <string>Settings...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>editBackupButt</sender>
   <signal>clicked()</signal>
   <receiver>mainwindow</receiver>
   <slot>editBackup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButt</sender>
   <signal>clicked()</signal>
   <receiver>mainwindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newBackupButt</sender>
   <signal>clicked()</signal>
   <receiver>mainwindow</receiver>
   <slot>newBackup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteBackupButt</sender>
   <signal>clicked()</signal>
   <receiver>mainwindow</receiver>
   <slot>deleteBackup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backupList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>mainwindow</receiver>
   <slot>checkSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cleanButt</sender>
   <signal>clicked()</signal>
   <receiver>mainwindow</receiver>
   <slot>cleanupBackup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keepRun</sender>
   <signal>clicked()</signal>
   <receiver>mainwindow</receiver>
   <slot>checkSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shutDown</sender>
   <signal>clicked()</signal>
   <receiver>mainwindow</receiver>
   <slot>checkSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settingstButton</sender>
   <signal>clicked()</signal>
   <receiver>mainwindow</receiver>
   <slot>aboutButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>editBackup()</slot>
  <slot>newBackup()</slot>
  <slot>deleteBackup()</slot>
  <slot>cleanupBackup()</slot>
  <slot>checkSelection()</slot>
  <slot>aboutButton()</slot>
 </slots>
</ui>
